modules/seqexec/server/src/main/scala/seqexec/server/SeqexecEngine.scala

```diff
@@ -617,117 +605,65 @@ object SeqexecEngine extends SeqexecConfiguration {
+
+      // More instruments to be added to the list here
+      val epicsInstruments = site match {
+        case Site.GS => List((conf.systemControl.f2, Flamingos2Epics), (conf.systemControl.gmos, GmosEpics), (conf.systemControl.gsaoi, GsaoiEpics))
+        case Site.GN => List((conf.systemControl.gmos, GmosEpics), (conf.systemControl.gnirs, GnirsEpics),
+          (conf.systemControl.niri, NiriEpics), (conf.systemControl.nifs, NifsEpics)
+        )
       }
-    } *> IO.apply(CaService.setIOTimeout(java.time.Duration.ofMillis(ioTimeout.toMillis)))
 
-    // More instruments to be added to the list here
-    val epicsInstruments = site match {
-      case Site.GS => List((f2Control, Flamingos2Epics), (gmosControl, GmosEpics), (gsaoiControl, GsaoiEpics))
-      case Site.GN => List((gmosControl, GmosEpics), (gnirsControl, GnirsEpics),
-        (niriControl, NiriEpics), (nifsControl, NifsEpics)
-      )
-    }
-    val epicsGaos = site match {
-      case Site.GS => List(gemsControl -> GemsEpics)
-      case Site.GN => List(altairControl -> AltairEpics)
-    }
-    val epicsSystems = epicsInstruments ++ List(
-      (tcsControl, TcsEpics),
-      (gwsControl, GwsEpics),
-      (gcalControl, GcalEpics)
-    ) ++ epicsGaos
-    val epicsInit: IO[List[Unit]] = caInit *> epicsSystems.filter(_._1.connect)
-      .map(x => initEpicsSystem(x._2, tops)).parSequence
-
-    def settings: IO[Settings] =
-        IO(LocalDate.now).map { now =>
-          Settings(site,
-                   odbHost,
-                   now,
-                   dhsServer,
-                   altairControl,
-                   gemsControl,
-                   dhsControl,
-                   f2Control,
-                   gcalControl,
-                   ghostControl,
-                   gmosControl,
-                   gnirsControl,
-                   gpiControl,
-                   gpiGdsControl,
-                   ghostGdsControl,
-                   gsaoiControl,
-                   gwsControl,
-                   nifsControl,
-                   niriControl,
-                   tcsControl,
-                   odbNotifications,
-                   instForceError,
-                   failAt,
-                   odbQueuePollingInterval,
-                   gpiUrl,
-                   ghostUrl,
-                   gpiGDS,
-                   ghostGDS)
-                 }
-
-    epicsInit *> settings
+      val epicsGaos = site match {
+        case Site.GS => List(conf.systemControl.gems -> GemsEpics)
+        case Site.GN => List(conf.systemControl.altair -> AltairEpics)
+      }
 
-  }
+      val epicsSystems = epicsInstruments ++ List(
+        (conf.systemControl.tcs, TcsEpics),
+        (conf.systemControl.gws, GwsEpics),
+        (conf.systemControl.gcal, GcalEpics)
+      ) ++ epicsGaos
+      val epicsInit: IO[List[Unit]] = caInit *> epicsSystems.filter(_._1.connect)
+        .map(x => initEpicsSystem(x._2, tops)).parSequence
```

Comment:
`.map(f).parSequence` → `.parTraverse(f)`

Reply:
I don't recall the source of this. But does this mean we'll do the init in parallel right? I'm a bit confused about the `parXXX` methods

Reply:
Yes, here the IO stuff will run in parallel.